test/libsolidity/smtCheckerTests/loops/for_loop_bmc_iterations_false_positive.sol

```diff
@@ -0,0 +1,16 @@
+contract C
+{
+	function f() public pure {
+		uint x = 0;
+		for (;;) {
+			x = 1;
+		}
+		assert(x == 1);
```

Comment:
Can you change this to something weird like `x == 1000` to make it clear that this is safe because it's unreachable?